app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,21 @@ def destroy
         end
     end
 
+    def edit
+        @post = Post.find(params[:id])
+    end
+
+    def update
+        @post = Post.find(params[:id])
+        if @post.update(post_params)
+            flash[:notice] = "Post Updated"
+            redirect_to post_path(@post)
+        else
+            flash[:error] = "Article was not Updated"
```

Comment:
The intent to let the user know there was an error updating post is correct. The error it's self isn't very useful for the user as they will not know why the updating the post failed.

This will give you a better understanding of how to get errors from your model instance -> https://api.rubyonrails.org/v6.1.4/classes/ActiveModel/Errors.html